src/metabase/models/recent_views.clj

```diff
@@ -340,27 +352,31 @@
 
 (defmethod fill-recent-view-info :table [{:keys [_model model_id timestamp model_object]}]
   (let [table model_object]
-    (when (and (mi/can-read? table) (true? (:active table)))
+    (when (and (mi/can-read? table)
+               ((complement #{:hidden "hidden"}) (:visibility_type table))
```

Comment:
This can be either a keyword or a string? Can we make it consistent?

Reply:
It's a keyword when called into from the query for popular_views, but yeah, we can clean it up